x-pack/plugins/upgrade_assistant/public/application/components/kibana_deprecations/kibana_deprecations.tsx

```diff
@@ -30,7 +30,8 @@ const i18nTexts = {
     defaultMessage: 'Kibana deprecation issues',
   }),
   pageDescription: i18n.translate('xpack.upgradeAssistant.kibanaDeprecations.pageDescription', {
-    defaultMessage: 'Resolve all critical issues before upgrading.',
+    defaultMessage:
+      'Resolve all critical issues before upgrading to the next major version of Elastic.',
```

Comment:
Again, but let's see what @debadair thinks, I'd put "to prepare the upgrade to"

Reply:
I don't think "to prepare" is necessary. I think it's understood that you're preparing to upgrade.